### ISSUE TITLE
Flutter Icons are not appearing in my web app

### ISSUE THREAD

**Issue by richasay** [flutter/flutter#40687]
## **Steps to Reproduce**
Here's the widget, in this case a drawer, where both the "hamburger" icon is not appearing, in Chrome I get a box placeholder, in FF I get a strange E! placeholder:

[main.dart.txt](https://github.com/flutter/flutter/files/3622533/main.dart.txt)
![missing_icons](https://user-images.githubusercontent.com/51710996/65061877-cd7e7400-d948-11e9-9850-177f6eec25e6.png)
```

  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        backgroundColor: Colors.lightBlue,
        title: Text('Port of Long Beach'),
      ),
      drawer: Drawer(
        child: ListView(
          children: <Widget>[
            DrawerHeader(
              decoration: BoxDecoration(
                gradient: LinearGradient(
                  colors: <Color>[
                    Colors.blue[300],
                    Colors.blue[700],
                  ]
                ),
              ),
              child: Text('Camera Manager'),
            ),
            CustomListTile(),
```

Here's the specific class - also included in the attached main.dart file:
```
class CustomListTile extends StatelessWidget {
  const CustomListTile({Key key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return InkWell(
      child: Row(
        children: <Widget>[
          Icon(Icons.camera_front),
          Text('Overview'),
          Icon(Icons.arrow_right),
        ],
      ),
    );
  }
}
```

Basically I cant get any icons to render in Flutter_Web - any advice would be great.  Its hard to pitch the benefits of this platform as the potential future, if something as simple as Icons cant be shown.

I added the fonts to the launcher as well, based on a previous search, no change.
[flutter_launcher_icons.yaml.txt](https://github.com/flutter/flutter/files/3622546/flutter_launcher_icons.yaml.txt)

**Comment by BondarenkoStas** [comment]
@richasay I see you use camera_front and arrow_right from material lib, can you please try #33231?
Please report about the result or if the problem is with another thing. There are several issues with material icons not rendered on flutter web.

**Comment by richasay** [comment]
Thanks! - I added the additional font URL and added the font to the /web/assets/ dir and they are now rendering!  Awesome!  @BondarenkoStas
